integration-tests/utils/browserstack.conf.js

```diff
@@ -70,6 +70,7 @@ const nightwatch_config = {
         os_version: '10',
         browserName: 'Chrome',
         browser_version: 'latest',
+        'browserstack.selenium_version' : '4.0.0-alpha-6'
```

Comment:
What is this for?

Reply:
https://raw.githubusercontent.com/SeleniumHQ/selenium/trunk/rb/CHANGES says `4.0.0-alpha-7` adds support for console messages in Chrome, but apparently it works in `alpha-6`.

Reply:
I'll update to alpha-7 anyways